test/e2e/utils/jira.ts

```diff
@@ -68,24 +68,24 @@ export const jiraAppUninstall = async (page: Page): Promise<Page> => {
-	await page.click("[data-testid='project-template-select-v2.ui.layout.screens.template-overview.template-overview-card.use-template-button.button']");
-	await page.click("[data-testid='project-template-select-v2.ui.layout.screens.project-types.footer.select-project-button-team-managed']");
-	await page.fill("[data-test-id='project-create.create-form.name-field.input'] input", projectId);
-	await page.fill("[data-test-id='project-create.create-form.advanced-dropdown.key-field.input-wrapper'] input", projectId);
-	await page.click("[data-test-id='project-create.create-form.create-screen.submit-button']");
+	// await page.click("[data-test-id='global-pages.directories.projects-directory-v2.create-projects-button.button.button']");
+	// await page.click("[data-testid='project-template-select-v2.ui.layout.category-overview.template-list-card.template-list-button'][aria-label='Kanban']");
+	// await page.click("[data-testid='project-template-select-v2.ui.layout.screens.template-overview.template-overview-card.use-template-button.button']");
+	// await page.click("[data-testid='project-template-select-v2.ui.layout.screens.project-types.footer.select-project-button-team-managed']");
+	// await page.fill("[data-test-id='project-create.create-form.name-field.input'] input", projectId);
+	// await page.fill("[data-test-id='project-create.create-form.advanced-dropdown.key-field.input-wrapper'] input", projectId);
+	// await page.click("[data-test-id='project-create.create-form.create-screen.submit-button']");
 
 	// V3 flow for future - apparently our e2e instance is still on v2
-	/*await page.click("[data-testid='global-pages.directories.projects-directory-v3.create-projects-button']");
+	await page.click("[data-testid='global-pages.directories.projects-directory-v3.create-projects-button']");
 	await page.click("[data-testid='project-template-select-v2.ui.layout.category-overview.template-list-card.template-list-button']");
 	await page.click("[data-testid='project-template-select-v2.ui.layout.screens.template-overview.template-overview-card.use-template-button.button']");
 	await page.click("[data-testid='project-template-select-v2.ui.layout.screens.project-types.footer.select-project-button-team-managed']");
 	await page.fill("[id='project-create.create-form.name-field.input']", projectId);
 	await page.click("[data-testid='project-access.field.button-trigger']");
 	await page.locator("[data-testid='project-access.field.dropdown-menu--content'] [data-testid='project-access.field.option']").nth(2).click();
 	await page.fill("[id='project-create.create-form.advanced-dropdown.key-field.input']", projectId);
-	await page.click("[data-test-id='project-create.create-form.create-screen.submit-button'] button");*/
+	await page.click("[data-test-id='project-create.create-form.create-screen.submit-button'] button");
```

Comment:
Looking at the screenshot, isn't the test id: `global-pages.directories.projects-directory-v3.create-projects-button`?

Reply:
Yup that's the change on line 80 and it's working now. 
Michel left two blocks of code, one for v2 and one for v3 (comment on line 79). I just uncommented the v3 and commented the v2
Tests are now failing on line 85 but the id looks the same in our Jira so I'm not too sure there.

Reply:
Gotcha. Do we know what element data-test-id='project-create.create-form.create-screen.submit-button is for?

Reply:
You can follow the whole flow of creating a project in Jira and that's what those ids are. This PR is not ready to be merged though bc it's still failing to find one of the buttons. I'll convert it to draft.

Reply:
There's a new page when creating a project, which asks to connect to other tools and thats messing up the flow with the old one. That's why its not working.